examples/python/invoke-rest-api/stack.yml

```diff
@@ -0,0 +1,17 @@
+provider:
+  name: openfaas
+  gateway: https://VEBA_FQDN_OR_IP
+functions:
+  restpost-fn:
+    lang: python3
+    handler: ./handler
+    image: vmware/veba-rest-post:latest
```

Comment:
Can you please align this with the function name?

Reply:
done